modules/accounts/accounts.js

```diff
@@ -1,189 +1,231 @@
     }
 
     getProviderAccounts(){    
         if (config.provider) {
-            return Helper.outputInfo(config.provider);
+            return Logger.info(config.provider);
         } else {
-            return Helper.outputInfo('No provider is set');
+            return Logger.warn('No provider is set');
         } 
     }
 
     getBalanceAccounts(account) {
         var pkh = account, f;
+        const tezosNode = config.provider;
         if (f = Helper.findKeyObj(config.identities, pkh)) {
             pkh = f.pkh;
         } else if (f = Helper.findKeyObj(config.accounts, pkh)) {
             pkh = f.pkh;
         } else if (f = Helper.findKeyObj(config.contracts, pkh)) {
             pkh = f.pkh;
         }
-        return eztz.rpc.getBalance(pkh).then(function(r){
-            return Helper.output(Helper.formatTez(r/100));
-        }).catch(function(e){
-            return Helper.outputError(e);
-        });
+        RpcRequest.fetchBalance(tezosNode, pkh);
     }
 
-    createTestnetAccount(args){
-        var pkh = args[0], f;  
-        if (Helper.findKeyObj(config.accounts, args[1])) 
-        return console.log(Helper.outputError('That account name is already in use'));
-        if (f = Helper.findKeyObj(config.identities, pkh)) {
-            return eztz.rpc.account(f, parseFloat(args[2]), true, true,f.pkh, 1400).then(function(r) {                  
-                var d=eztz.contract.hash(r.hash, 0);        
-                config.accounts.push({
-                    label : args[1],
-                    pkh : d,
-                    identity : pkh,        
-                });
-                jsonfile.writeFile(confFile, config);
-                return Helper.output('New account created ' + args[1]);
-            }).catch(function(e) {           
-                return Helper.outputError(e);
-            });
-        } else {
-            return Helper.outputError(pkh + ' is not a valid identity');
+    async createTestnetAccount(args) {
+        var accountLabel = args[0];
+        const conseiljs = require(CONSEIL_JS);
+        const keys = this.getKeys(accountLabel);
+        if(keys) {
+            return Logger.error(`Account with this label already exists.`);
         }
-    }
 
-    async loadTezsterConfig() {
-        eztz = require('../../lib/eztz.cli.js').eztz;
-        const jsonfile = require('jsonfile');
-        config = jsonfile.readFileSync(confFile);
-        if (config.provider) {
-            eztz.node.setProvider(config.provider);
-        }  
-        const _sodium = require('libsodium-wrappers');
-        await _sodium.ready;
-        eztz.library.sodium = _sodium;
+        try {
+            const mnemonic = conseiljs.TezosWalletUtil.generateMnemonic();
+            const keystore = await conseiljs.TezosWalletUtil.unlockIdentityWithMnemonic(mnemonic, '');
+            this.addIdentity(accountLabel, keystore.privateKey, keystore.publicKey, keystore.publicKeyHash, '');
```

Comment:
You need to display mnemonic to user and ask them to store it for future reference. 
also it's not a faucet account, it's a wallet.
display log messages accordingly.